Assets/Project/Modules/VFX/Testing/Scripts/VFXTester.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Popeye.Core.Services.ServiceLocator;
+using Popeye.Modules.VFX.Generic;
+using Popeye.Modules.VFX.ParticleFactories;
+using Unity.Mathematics;
+using UnityEngine;
+
+public class VFXTester : MonoBehaviour
+{
+    [SerializeField] private ParticleTypes _testParticleType1;
+    [SerializeField] private Transform _particleParent;
+    
+    private void Start()
+    {
+        StartCoroutine(TestVFX());
+    }
+
+    private IEnumerator TestVFX()
```

Comment:
You can also add the NaughtyAttributes header [Button()] on top of a function to execute it